src/skmultiflow/meta/adaptive_random_forest_regressor.py

```diff
@@ -0,0 +1,497 @@
+        Number of trees in the ensemble.
+
+    max_features : int, float, string or None, optional (default="auto")
+        Max number of attributes for each node split.
+
+        - If int, then consider ``max_features`` features at each split.
+        - If float, then ``max_features`` is a percentage and
+          ``int(max_features * n_features)`` features are considered at
+          each split.
+        - If "auto", then ``max_features=sqrt(n_features)``.
+        - If "sqrt", then ``max_features=sqrt(n_features)`` (same as "auto").
+        - If "log2", then ``max_features=log2(n_features)``.
+        - If None, then ``max_features=n_features``.
+
+    disable_weighted_vote: bool, optional (default=False)
+        Weighted vote option.
+
+    lambda_value: int, optional (default=6)
+        The lambda value for bagging (lambda=6 corresponds to Leverage Bagging).
+
+    drift_detection_method: BaseDriftDetector or None, optional
+        (default=ADWIN(0.001))
+        Drift Detection method. Set to None to disable Drift detection.
+
+    warning_detection_method: BaseDriftDetector or None, default(ADWIN(0.01))
+        Warning Detection method. Set to None to disable warning detection.
+
+    max_byte_size: int, optional (default=33554432)
+        (`RegressionHoeffdingTree` parameter)
+        Maximum memory consumed by the tree.
+
+    memory_estimate_period: int, optional (default=2000000)
+        (`RegressionHoeffdingTree` parameter)
+        Number of instances between memory consumption checks.
+
+    grace_period: int, optional (default=50)
+        (`RegressionHoeffdingTree` parameter)
+        Number of instances a leaf should observe between split attempts.
+
+    split_confidence: float, optional (default=0.01)
+        (`RegressionHoeffdingTree` parameter)
+        Allowed error in split decision, a value closer to 0 takes longer
+        to decide.
+
+    tie_threshold: float, optional (default=0.05)
+        (`RegressionHoeffdingTree` parameter)
+        Threshold below which a split will be forced to break ties.
+
+    binary_split: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, only allow binary splits.
+
+    stop_mem_management: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, stop growing as soon as memory limit is hit.
+
+    remove_poor_atts: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, disable poor attributes.
+
+    no_preprune: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, disable pre-pruning.
+
+    leaf_prediction: string, optional (default='nba')
+        (`RegressionHoeffdingTree` parameter)
+        Prediction mechanism used at leafs.
+
+        - 'mc' - Majority Class
+        - 'nb' - Naive Bayes
+        - 'nba' - Naive Bayes Adaptive
+
+    nb_threshold: int, optional (default=0)
```

Comment:
We could add an observation here telling that this parameter does not apply in the regression case. What do you think @jacobmontiel? We could also extend this to the RegressionHoeffdingTree.

Reply:
Agree for the Adaptive Random Forest Regressor implementation. In the case of `RegressionHoeffdingTree` currently, there are two options available `perceptron` and `mean`.

Reply:
I think I marked the wrong line. I refer to the `nb_threshold`. Could we add some note in the documentation?

Reply:
What should we do with that one? Mark them as not used in the documentation, & maybe issue a warning if it is set by the user?

Reply:
Since this is not relevant to the regression case I vote to omit it completely from the signature and the documentation.

Reply:
I agree with that solution!

Reply:
Ok, I cleaned that up.